repose-aggregator/components/filters/rackspace-identity-basic-auth/src/main/scala/org/openrepose/filters/rackspaceidentitybasicauth/RackspaceIdentityBasicAuthFilter.scala

```diff
@@ -100,19 +98,34 @@ class RackspaceIdentityBasicAuthFilter @Inject()(configurationService: Configura
   }
 
   override def doFilter(servletRequest: ServletRequest, servletResponse: ServletResponse, filterChain: FilterChain) {
-    new FilterLogicHandlerDelegate(servletRequest, servletResponse, filterChain).doFilter(this)
+    val wrappedHttpRequest = new HttpServletRequestWrapper(servletRequest.asInstanceOf[HttpServletRequest])
+    val httpResponse = servletResponse.asInstanceOf[HttpServletResponse]
+
+    val filterAction = handleRequest(wrappedHttpRequest, httpResponse)
+    filterAction match {
```

Comment:
I figured that was the case. I am not opposed to using that FilterAction and dropping NOT_SET. Perhaps a future story to do that should be created?